src/fastoad/models/performances/mission/openmdao/tests/data/test_mission_takeoff.yml

```diff
@@ -0,0 +1,229 @@
+phases:
+  taxi_out:
+    parts:
+      - segment: start
+        target:
+          altitude:
+            value: 1000
+            unit: m
+      - segment: taxi
+        isa_offset: 0.0
+        thrust_rate: data:mission:operational:taxi_out:thrust_rate
+        true_airspeed:
+          value: data:mission:operational:taxi_out:true_airspeed
+          default: 0.
+        target:
+          time: data:mission:operational:taxi_out:duration
+      - segment: mass_input
+        target:
+          mass: data:mission:operational:TOW
+  takeoff:
+    engine_setting: takeoff
+    polar:
+      CL: data:aerodynamics:aircraft:takeoff:CL
+      CD: data:aerodynamics:aircraft:takeoff:CD
+      ground_effect: Raymer
+      CL0_clean: data:aerodynamics:aircraft:takeoff:CL0_clean
+      CL_alpha:
+        value: data:aerodynamics:aircraft:takeoff:CL_alpha
+        unit: 1/rad
+      CL_high_lift: data:aerodynamics:high_lift_devices:takeoff:CL
+    thrust_rate: 1.0
+    isa_offset: data:mission:operational:ISA_offset
+    parts:
+      - segment: ground_speed_change
+        target:
+          equivalent_airspeed:
+            value: data:mission:operational:takeoff:Vr
+      - segment: rotation
+        target:
+          delta_altitude:
+            value: 35
+            unit: ft
+      - segment: end_of_takeoff
+        time_step: 0.05
+        target:
+          delta_altitude:
+            value: 35
+            unit: ft
```

Comment:
I understand the need to implement 3 segments. But should we not propose only one, that would bundle the 3 parts, for the mission file? I know this is currently not possible, but if makes more sense, we could make it possible.
Also, why `rotation` and `end_of_takeoff` have the same target?

Reply:
About the last question, the rotation segment stops when the aircraft lifts off. At least this is the criterion to stop the segment since the reaction force from the ground disappears, it is necessary to change the equations and so change segment. I could not translate this criterion into a target, so it is set to 35ft arbitrarily without perturbing the takeoff simulation.
About the first remark, it would make more sense for the user indeed